### PR TITLE
Update preview-channel.md

### PR DESCRIPTION
**Pull request by asklar**

### CODE REVIEW

**Comment by PRMerger7** [comment]
@asklar : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by zaryaf — APPROVED** [review]

**Comment by zaryaf** [comment]
@mcleanbyron Should Alex change his base branch to master or is that something we can do?

**Comment by asklar** [comment]
you should be able to merge from this branch without issues

File: hub/apps/windows-app-sdk/preview-channel.md [hunk 1]
```diff
@@ -49,10 +49,17 @@ To get your machine back into a good state, take the following steps:
...
 
 ```Powershell
-# This script must be run from an elevated PowerShell window (right-click PowerShell in the Start menu, and select Run as Administrator).
+# This script must be run from an elevated Windows PowerShell window (right-click PowerShell in the Start menu, and select Run as Administrator).
+
+# Remove the Windows App SDK 1.0 Preview1/2, and all apps that use it.
+
+$winappsdk = "Microsoft.WindowsAppRuntime.1.0-preview*"
+Get-AppxPackage | Where-Object { $_.Dependencies -like $winappsdk } | Remove-AppxPackage
+Remove-AppxPackage $winappsdk
```
Comment (asklar) [inline hub/apps/windows-app-sdk/preview-channel.md]:
```suggestion
Get-AppxPackage $winappsdk | Remove-AppxPackage
```

**Comment by PRMerger12** [comment]
@asklar : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**